workflow/rules/regions.smk

```diff
@@ -0,0 +1,47 @@
+rule build_sample_regions:
+    input:
+        get_varlociraptor_preprocessing_input,
```

Comment:
As the name says this was previously only used for getting varlociraptor input.
To prevent confusion this function should probably get a more general name.

Reply:
Fixed with commit 93bd785.